tests/vppinputasync.h

```diff
@@ -28,18 +28,20 @@ class VppInputAsync : public VppInput
 
     bool read(SharedPtr<VideoFrame>& frame);
 
+    /*startupSize means how many buffers will preload to memory in startup, it will overide queueSize*/
     static SharedPtr<VppInput>
-    create(const SharedPtr<VppInput>& input, uint32_t queueSize);
+    create(const SharedPtr<VppInput>& input, uint32_t queueSize, uint32_t startupSize = 0);
```

Comment:
Is it possible to make m_startupSize public to allow direct init/read/modifiy of m_startupSize since the value never changes once it's initialized.  This will eliminate the need for introducing startupSize in every function.

Having two separate variables for the same purpose whereby the values never change requires additional maintenance.